.gitlab-ci.yml

```
@@ -205,11 +205,14 @@ test-lin-dev-clang-cran:
 
 # stated dependency on R
 test-lin-ancient-cran:
-  image: registry.gitlab.com/jangorecki/dockerfiles/r-3.3.0
   <<: *test-lin
+  image: registry.gitlab.com/jangorecki/dockerfiles/r-3.3.0
+  variables:
+    _R_CHECK_FORCE_SUGGESTS_: "FALSE"
   script:
     - *install-deps
-    - R CMD check --no-manual $(ls -1t data.table_*.tar.gz | head -n 1)
+    - Rscript -e 'install.packages("https://cran.r-project.org/src/contrib/Archive/xts/xts_0.12.2.tar.gz", repos = NULL, type = "source")'
+    - R CMD check --no-manual --no-build-vignettes --ignore-vignettes $(ls -1t data.table_*.tar.gz | head -n 1)
```

Conceivably, we'll catch up to an R version where knitr can be installed again in the future, so leave a note here IMO.

```suggestion
    # knitr requires evaluate, which requires R 3.6.0.
    # Restore checking vignettes if upgrading our R dependency means knitr can be installed.
    - R CMD check --no-manual --no-build-vignettes --ignore-vignettes $(ls -1t data.table_*.tar.gz | head -n 1)
```